lib/matobjnz.gi

```diff
@@ -1640,3 +1640,20 @@ local i, n, ords, base, imat, vec, one,cp,op,zero,fam;
     Info(InfoMatrix,1,"Characteristic Polynomial returns ", cp);
     return cp;
 end );
+
+
+##
+
+InstallOtherMethod( DegreeFFE, "for IsZmodnZVectorRep",
+    [ IsZmodnZVectorRep ],
+function(vec)
+    # TODO: check that modulus is a prime
```

Comment:
That implementation is missing.

Reply:
Maybe a bit simpler:
```
InstallOtherMethod( DegreeFFE, ["IsZmodnZVectorRep"], v-> 1);
```

Reply:
Not sure what you mean? The fact there is an unresolved TODO here?

Well, it's a TODO for future work, the current code also is not performing this check, and in principle one may wish to do this not in that function but e.g. when constructing the matrix, and then carrying around the info in an `isprime` value... perhaps the modulus and the `isprime` bool could also be stored in the type or family... I did not want to tackle these decisions here, and considering that `DegreeFFE` does not specify what it does for an invalid argument, I figure returning `1` here while not great is also not the worst...

Reply:
Thank for the explanation! I was simply not sure if that was an oversight and wanted to make sure.

Reply:
Sure! I appreciated it as such, just wasn't sure if I understood the question right :-)